### PR TITLE
[Bug Fix] Harm Touch Critical Ratio

### PR DESCRIPTION
**Pull request by fryguy503**
This is used to tune and bring HT Crits more in line with the correct values as proven in screenshots.

### CODE REVIEW

**Comment by Akkadius** [comment]
What screenshots?

File: zone/effects.cpp [hunk 1]
```diff
@@ -96,6 +95,10 @@ int64 Mob::GetActSpellDamage(uint16 spell_id, int64 value, Mob* target) {
 			ratio += RuleI(Spells, WizCritRatio); //Default is zero
 		}
 
+		if (IsClient() && (spell_id == SPELL_HARM_TOUCH || spell_id == SPELL_HARM_TOUCH2 || spell_id == SPELL_IMP_HARM_TOUCH )) {
```
Comment (Akkadius) [inline zone/effects.cpp:98]: There's a few places in the code where we are doing this

```
spell_id == SPELL_HARM_TOUCH || spell_id == SPELL_HARM_TOUCH2 || spell_id == SPELL_IMP_HARM_TOUCH
```

We can simplify this to a helper

```cpp
bool Mob::IsHarmTouchSpell(uint16 spell_id)
{
	return spell_id == SPELL_HARM_TOUCH ||
		   spell_id == SPELL_HARM_TOUCH2 ||
		   spell_id == SPELL_IMP_HARM_TOUCH;
}
```

And use it in place

**Review by Akkadius — COMMENTED** [review]
Minor comment I'd like to see addressed, otherwise LGTM

**Comment by fryguy503** [comment]
> What screenshots?

image

In era showing higher crits

**Review by Akkadius — APPROVED** [review]

**Pull request merged**